src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1897,10 +1897,7 @@ private function registerCacheConfiguration(array $config, ContainerBuilder $con
         if (isset($config['prefix_seed'])) {
             $container->setParameter('cache.prefix.seed', $config['prefix_seed']);
         }
-        if ($container->hasParameter('cache.prefix.seed')) {
-            // Inline any env vars referenced in the parameter
-            $container->setParameter('cache.prefix.seed', $container->resolveEnvPlaceholders($container->getParameter('cache.prefix.seed'), true));
```

Comment:
should be reverted IMHO - I'm personally not convinced it's a good idea to compute the namespace at runtime